packages/cos.go

```diff
@@ -0,0 +1,67 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package packages
+
+import (
+	"fmt"
+	"strconv"
+
+	"cos.googlesource.com/cos/tools.git/src/pkg/cos"
+	"github.com/GoogleCloudPlatform/osconfig/osinfo"
+)
+
+// COSPackageInfoExists returns whether COS package information exists.
+func COSPackageInfoExists() bool {
```

Comment:
Leave this as a var in packages.go and populated it in an init() function in this file. That way this check is only done once not every run, and it's a bit simpler to test.

Reply:
Done in the second commit.